src/internet_identity/tests/integration/http.rs

```diff
@@ -609,9 +609,9 @@ fn should_list_aggregated_session_seconds_and_event_data_counters() -> Result<()
     api::prepare_delegation(
         &env,
         canister_id,
-        test_authn_method().principal(),
+        authn_method_ic0.principal(),
```

Comment:
Why this change?

Reply:
So that the counter of `"https://some-dapp-2.com"` is not just 1, but a higher number so that it couldn't just be an off by one error 😉

Reply:
Ok thanks for the explanation.